libraries/ts-command-line/src/test/__snapshots__/CommandLineParameter.test.ts.snap

```diff
@@ -556,15 +644,15 @@ a longer description
 
 Optional arguments:
   -h, --help            Show this help message and exit.
-  -c {one,two,three,default}, --choice {one,two,three,default}
+  -c {one,two,three,choice with spaces,default}, --choice {one,two,three,choice with spaces,default}
```

Comment:
The help docs should really be quoting anything that needs to be quoted.

Reply:
We should fix that separately.